include/swift/Driver/ToolChain.h

```diff
@@ -195,6 +195,8 @@ class ToolChain {
   /// relative to the compiler.
   void getRuntimeLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
                              const llvm::opt::ArgList &args, bool shared) const;
+  void getRuntimeLibraryPathWithArch(SmallVectorImpl<char> &runtimeLibPath,
+                             const llvm::opt::ArgList &args, bool shared) const;
```

Comment:
Nitpick: the indentation is supposed to match the open paren (while wrapping at 80 columns).